src/modules/system-settings/domain/services/system-settings.service.ts

```
@@ -0,0 +1,145 @@
+import { Injectable, Inject } from '@nestjs/common';
+import { EventEmitter2 } from '@nestjs/event-emitter';
+import { SystemSettings, SystemSettingsData } from '../entities/system-settings.entity';
+import { ISystemSettingsRepository } from '../interfaces/system-settings-repository.interface';
+import { DefaultSettingsService } from './default-settings.service';
+import { SystemSettingsNotFoundException } from '../exceptions/system-settings.exceptions';
```

ESLint

[error] 'SystemSettingsNotFoundException' is defined but never used. Allowed unused vars must match /^_/u.

(@typescript-eslint/no-unused-vars)